tests/testthat/test-derive_param_tte.R

```diff
@@ -640,7 +640,7 @@ test_that("derive_param_tte Test 9: errors if set_values_to contains invalid exp
       source_datasets = list(adsl = adsl, ae = ae),
       set_values_to = exprs(
         PARAMCD = paste0("TTAE", as.numeric(as.factor(AEDECOD))),
-        PARAM = past("Time to First", AEDECOD, "Adverse Event"),
+        PARAM = paste("Time to First", AEDECOD, "Adverse Event"),
```

Comment:
Please don't change this. The mistake is intentional to cause an error. Now the test is failing because the is no error anymore.

Reply:
I see. I thought I had changed this by mistake. I'll change it back. Thanks.